README.md

```diff
@@ -7,173 +7,61 @@
 [![Mattermost](https://badgen.net/badge/Mattermost/Outline%20Community/blue)](https://community.internetfreedomfestival.org/community/channels/outline-community)
 [![Reddit](https://badgen.net/badge/Reddit/r%2Foutlinevpn/orange)](https://www.reddit.com/r/outlinevpn/)
 
-This is the repository to hold the future Outline SDK as we develop it. The goal is to clean up and move reusable code from [outline-ss-server](https://github.com/Jigsaw-Code/outline-ss-server) and [outline-go-tun2socks](https://github.com/Jigsaw-Code/outline-go-tun2socks).
+<p align="center">
+<img src="https://github.com/Jigsaw-Code/outline-brand/blob/main/assets/powered_by_outline/color/logo.png?raw=true" width=400pt />
+</p>
 
-**WARNING: This code is not ready to be used by the public. There's no guarantee of stability.**
+> **Warning**
+> This code is not ready to be used by the public. There's no guarantee of stability.
 
-Tentative roadmap:
+The Outline SDK helps developers:
+- Create tools to protect against network-level interference
+- Add network-level interference protection to existing apps, such as content or communication apps
+
+## Advantages
+
+| Multi-Platform | Proven Technology | Composable |
+|-|-|-|
+| The Outline SDK can be used on Android, iOS, Windows, macOS or Linux. | The Outline Client and Server have been using the SDK for years, helping millions of users in the harshest conditions. | The SDK interfaces were carefully designed to allow for composition and reuse, so you can craft your own transport. |
```

Comment:
> ... have been using the SDK for years ...

Are we gonna really mention "for years" here 😄 ? Because the SDK repository has only been created less than one year.

Reply:
I updated to "The Outline Client and Server have been using the code in the SDK for years" to be clear it's the code, not the SDK